samcli/local/apigw/local_apigw_service.py

```diff
@@ -327,6 +329,29 @@ def _query_string_params(flask_request):
 
         return query_string_dict
 
+    @staticmethod
+    def _multi_value_headers(event_headers):
+        """
+        Constructs an APIGW equivalent multi-value header dictionary
+
+        Parameters
+        ----------
+        event_headers event headers
+            Request Headers
+
+        Returns dict (str: list of str)
+        -------
+            Returns a dictionary of key to list of strings
+
+        """
+        header_dict = {}
+
+        # Multi-value request headers is not really supported by Flask.
```

Comment:
[This](https://github.com/pallets/werkzeug/blob/master/werkzeug/datastructures.py#L910) indicates otherwise. Are we sure it is not supported?